README.md

```diff
@@ -1,7 +1,7 @@
 Core ML Community Tools
 =======================
 
-Core ML community tools contains all supporting tools for CoreML model
+Core ML community tools contains all supporting tools for Core ML model
 conversion and validation. This includes Scikit Learn, LIBSVM, Caffe,
```

Comment:
Also, I think the official product name of [scikit-learn](https://en.wikipedia.org/wiki/Scikit-learn) is just "scikit-learn" (all lowercase with a dash). It can be corrected in a separate PR.

Reply:
@rxwei Thanks for the PR!